src/Functions/toMillisecond.cpp

```diff
@@ -0,0 +1,20 @@
+#include <Functions/FunctionFactory.h>
+#include <Functions/DateTimeTransforms.h>
+#include <Functions/FunctionDateOrDateTimeToSomething.h>
+#include <DataTypes/DataTypesNumber.h>
+
+
+namespace DB
+{
+
+using FunctionToMillisecond = FunctionDateOrDateTimeToSomething<DataTypeUInt8, ToMillisecondImpl>;
```

Comment:
Why is it UInt8? It also seems like `ToMillisecondImpl` is missing

Reply:
I am pretty new to open source contributions so please bear with my stupid questions. I just found it should be UInt16 I'll fix it right away, as the "ToMillisecondImpl is missing" I didn't quite understand that so if you can explain it to me it would be great. Like Is that something I've to implement? because for toSecond() I couldn't find anything related to TosecondImpl

Reply:
> Like Is that something I've to implement?

Yes. `ToSecondImpl`: https://github.com/ClickHouse/ClickHouse/blob/f7cf512824306fb53866cf41a35ab9bd5de041f3/src/Functions/DateTimeTransforms.h#L1460-L1483